lib/builder/webpack/base.js

```diff
@@ -40,6 +40,7 @@ export default class WebpackBaseConfig {
       ]
     }
 
+    delete options.exclude
```

Comment:
What babel does itself introduce an `exclude` option? I mean if this is something specific to nuxt it may be flattened into `nuxt.build.exclude` or something like that :)